tests/integration/conftest.py

```diff
@@ -98,3 +99,10 @@ def _get_osname():
             if line.startswith("PRETTY_NAME="):
                 return line.split("=", maxsplit=1)[1].strip().strip('"')
     return ""
+
+
+def download_NM_plugin(plugin):
```

Comment:
Please use lowercase chars, even for `nm`.
But based on what the function does, I do not understand how it related to `NM` at all.
It just installs a package.
Therefore, it may fit something like: `install_package`.

Reply:
As it is fixture now, I have renamed it to `download_nm_team_plugin()`. Thanks!